File: r/page-012.R
###
### Practical Statistics for Data Sceintists
###


state <- read.csv("D:/my-coding/practical-statistics-4-ds-book/data/state.csv")
state
state.mean<-mean(state[['Population']])
state.mean
state.mean.trimmed_1<-mean(state[['Population']], trim=0.1)
state.mean.trimmed_1
state.median=median(state[['Population']])
state.median

File: python/page-012.py
###
### Practical Statistics for Data Sceintists
###


import pandas as pd
from scipy import stats

state = pd.read_csv('D:/my-coding/practical-statistics-4-ds-book/data/state.csv')
state_mean = state['Population'].mean()
print ( state_mean )
state_mean_trimmed_01 = stats.trim_mean(state['Population'], 0.1)
print ( state_mean_trimmed_01 )
state_median = state['Population'].median()
print ( state_median )

